exercises/random/challenge-057.py

```diff
@@ -0,0 +1,31 @@
+### RANDOM CHALLENGE ###
+
+"""
+057
+Update program 056 so that it tells the user
+if they are too high or low before they pick again.
+"""
+
+# Play around to understand more: added a different message if user guess doesn't match.
+import random
+import sys
+
+computer_choice = random.randint(1, 10)
+correct = False
+
+
+def check_user_choice():
+
+    while correct == False:
+        user_choice = int(input("Pick a number between 1 and 10: "))
+        if user_choice == computer_choice:
+            correct == True
+            print("You rock! Computer choice matched.", int(computer_choice))
+            sys.exit()
```

Comment:
In a real problem you should only `sys.exit` from `__main__` or you really want to stop the entire program. In this case the function should yield back to main and main should probably terminate by reaching the end of script rather than aborting completely.

This is because a function that can terminate all execution cannot be safely reused.

Reply:
replaced sys with return statement.